src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/LocalAppContextSwitches.cs

```diff
@@ -30,7 +30,7 @@ static LocalAppContextSwitches()
             catch (Exception e)
             {
                 // Don't throw an exception for an invalid config file
-                SqlClientEventSource.Log.TryTraceEvent("<sc.{0}.{1}|INFO>: {2}", TypeName, MethodBase.GetCurrentMethod().Name, e);
+                SqlClientEventSource.Log.TryTraceEvent("<sc.{0}.{1}|INFO>: {2}", TypeName, "..ctor", e);
```

Comment:
shouldn't this be `.cctor` for a static constructor?

Reply:
Also, can you inline the TypeName, it's a nameof in a variable which is a bit pointless when it's single use.